docker/Running using Docker.md

```diff
@@ -0,0 +1,34 @@
+# Running Using Docker
+
+This Dockerfile is used to build a docker container for Archipelago (currently 0.4.4) that fulfills all the requirements in [Running from Source](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/running%20from%20source.md). The end user ideally just needs to download the container and configure a few things and the Webhost will run as expected.
```

Comment:
I wonder if there's a way to always point it at the latest release, so we don't have to update the version in these files manually every time.

If we do have to manually update the version, I think it would be better to only need to change it in one place for the Dockerfile.

In this line we could say "(latest release)" instead of the version number.
And in the line down below, it could just say "Pull the source of Archipelago to /Archipelago"

Then the only place we would need to update is in the `Dockerfile` itself.

Reply:
This could be managed using ADD in the dockerfile. The dockerfile would then look like this
```suggestion
FROM python:3.11.8

#clone archipelago
RUN mkdir -p /Archipelago
ADD .. /Archipelago
```

Only reason I did the clone is WebHost for the bleeding edge of Archipelago can have bugs. Maybe we could use tags to automate container creation when a new tag is created.